plugins/woocommerce/tests/php/src/Internal/DataStores/Reviews/ReviewVerificationQueryTest.php

```
@@ -0,0 +1,216 @@
+				}
+			}
+		);
+		$this->assertSame( 0, $badge_queries, 'Verified-owner badge should not query after the batched prime.' );
+	}
+
+	/**
+	 * The batched result must be byte-identical to the per-review (unbatched) path.
+	 */
+	public function test_prime_matches_per_review_path(): void {
+		// Unbatched baseline via the filter gate.
+		list( $product_a, $expected_a ) = $this->build_verified_owner_fixture();
```

PHPMD (2.15.0)

[warning] Avoid unused local variables such as '$product_a'. (undefined)

(UnusedLocalVariable)